tools/license_header.py

```diff
@@ -68,6 +68,9 @@
                # Git submodules under different licenses
                '3rdparty',
 
+               # 3rdparty headerfiles under different licenses
+               'include/mkldnn',
```

Comment:
Can you explain? MKL-DNN is also under Apache-2.0.

Reply:
I imagine mkldnn would be licensed to intel instead of ASF even though the license is apache 2.0

Reply:
license check would fail if this is not whitelisted.